server/views.py

```diff
@@ -19,7 +19,10 @@ def get_user_info_dict(user):
 def index(request):
     user_info = get_user_info_dict(request.user)
     notebooks = (
-        Notebook.objects.annotate(latest_revision=Max("revisions__created"))
+        Notebook.objects.annotate(
+            latest_revision=Max("revisions__created"), num_revisions=Count("revisions")
+        )
+        .filter(num_revisions__gte=10)
```

Comment:
Could you change this to a constant like `MIN_FIREHOSE_REVISIONS` in `server/settings.py`?